action.yml

```diff
@@ -1,108 +1,19 @@
-name: Pros Build
-description: Automatically build pros projects
+name: 'ABUCKY0 PROS Build'
+description: 'Automatically build PROS templates'
+branding:
+  color: 'blue'
+  icon: 'box'
+  colorized: true
 
 inputs:
-  repository:
-    description: Target repository
+  multithreading:
     required: false
-    default: ${{ github.repository }}
-  library-path:
+    default: true
+    description: Whether to use multithreading or not
```

Comment:
Multithread what? This description tells me nothing that I didn't already know from using this option.

Reply:
The build process, I’ll update that. Basically just the difference between make quick and make quick -j